test/cookbooks/test/recipes/rubygems_url.rb

```diff
@@ -0,0 +1,5 @@
+chef_client_updater 'Install latest Chef' do
+  version '13.6.0'
+  rubygems_url 'https://www.rubygems.org'
```

Comment:
that feels like a not super useful test for the time it may consume?

if you no-op all the code behind `rubygems_url` and break it completely it'll still pass here...

i don't have any good suggestions for how to test it that aren't awful so kinda leaning towards suggesting not testing it...

adding chefspec testing that the resource supports rubygems_url might be more useful to just flame test that this argument is here, but since we don't already have chefspec testing i wouldn't want to burden this PR with delivering that...

Reply:
I'll try to come up with a better test.